rdmo/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3081,6 +3464,19 @@ msgstr ""
 msgid "(optional)"
 msgstr ""
 
+#: projects/templates/projects/project_questions_question_warning.html:6
+msgid ""
+Warning: Multiple answers exist in the database. This can happen due some 
+technical problem in the past. You can remove unwanted answers using the 
+\Remove item\" field (<i class=\"fa fa-times\"></i>). Please contact "
+support if the problem persists.
```

Comment:
```suggestion
"Warning: Multiple answers exist in the database. This can happen due some "
"technical problem in the past. You can remove unwanted answers using the "
"\"Remove item\" field (<i class=\"fa fa-times\"></i>). Please contact "
"support if the problem persists."
```